Snakefile

```diff
@@ -181,6 +187,17 @@ rule extract_raw:
     shell:
         "unzip {input} raw/{wildcards.dirname}/{wildcards.filename} -d {DATA}"
 
+# Get emoji data directly from Unicode CLDR
```

Comment:
A nitpick: should this comment appear above `download_unicode_data` instead of here?